docs/architecture/adt-014-proportional-slashing.md

```diff
@@ -0,0 +1,87 @@
+
+```
+slash_amount = power // power is the faulting validator's voting power.
+```
+
+However, this will incentivize validators with large amounts of stake to split up their voting power amongst accounts, so that if they fault, they all get slashed at a lower percent.  The solution to this is to take into account not just a validator's own voting percentage, but also the voting percentage of all the other validators who get slashed in a short time window.
+
+```
+slash_amount = (power_1 + power_2 + ... + power_n) // where power_i is the voting power of the ith validator faulting in the period
+```
+
+Now, if someone splits a validator of 10% into two validators of 5% each which both fault, then they both fault in the same window, they both will still get slashed at the sum 10% amount.
+
+However, an operator might still choose to split up their stake across multiple accounts with hopes that if any of them fault independently, they will not get slashed at the full amount.  In the case that the validators do fault together, they will get slashed the same amount as if they were one entity.  There is no con to splitting up.  However, if operators are going to split up their stake without actually decorrelating their setups, this also causes a negative externality to the network as it fills up validator slots that could have gone to others or increases the commit size.  In order to disincentivize this, we want it to be the case such that splitting up a validator into multiple validators and they fault together is punished more heavily that keeping it as a single validator that faults.
+
+We can achieve this by not only taking into account the sum of the percentages of the validators that faulted, but also the *number* of validators that faulted in the window.  One general form for an equation that fits this desired property looks like this:
+
+```
+slash_amount = (sqrt(power_1) + sqrt(power_2) + ... + sqrt(power_n))^2
+```
+
+So now, for example, if one validator of 10% faults, it gets a 10% slash, while if two validators of 5% each fault together, they both get a 20% slash ((sqrt(0.05)+sqrt(0.05))^2).
+
+One will note, that this model doesn't differentiate between multiple validators run by the same operators vs validators run by different operators.  This can be seen as an additional benefit in fact.  It incentivizes validators to differentiate their setups from other validators, to avoid having correlated faults with them or else they risk a higher slash.  So for example, operators should avoid using the same popular cloud hosting platforms or using the same Staking as a Service providers.  This will lead to a more resilient and decentralized network.
+
+
+We can allow some parameterization by multiplying the slash by an on-chain governable parameter.
+
+```
+slash_amount = k * (sqrt(power_1) + sqrt(power_2) + ... + sqrt(power_n))^2  // where k is an on-chain parameter for this specific slash type
```

Comment:
Griefing (intentionally being slashed to make another's slash worse) could be a concern here.

Reply:
Yeah, the idea of incorporating the inverse gini somehow was meant to mitigate griefing attacks using many trivial size validators (1 atom).  However, I'm not over concerned at the moment, because, from playing around with it on paper, it's hard to increase the slash by much.  Also, because our validator set is saturated anyways, with a pretty high minimum, its not a huge concern, you'd have to slash yourself quite a bit in order to grief.  The larger the grief, the griefer loses quadratically more, so I think it's okay.

Reply:
I agree that in practice it is unlikely to be a concern but I think we should mention it, and include that analysis. For some exponents or values `k` it might be a concern.

Reply:
Updated ADR to make note of griefing possibilities.